class NotFoundException(Exception):
    pass


class NotCompletedException(Exception):
    pass
